api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb

```diff
@@ -19,7 +19,7 @@ def timestamp
   end
 
   def parent_organization
-    @parent_organization ||= current_user.organizations.to_a.find { |o| o.id.to_s == params[:organization_id].to_s }
+    @parent_organization ||= MnoEnterprise::Organization.find_one(params[:organization_id], :orga_relations)
```

Comment:
I think this helper was not used before and is just a bad copy paste  from the regular jpi controller.
How about removing it and inline the code in the controller that uses it?